diff -burN mtx-1.3.3.orig/Makefile.in mtx-1.3.3/Makefile.in
--- mtx-1.3.3.orig/Makefile.in	Fri Feb 21 18:56:28 2003
+++ mtx-1.3.3/Makefile.in	Tue Jul 22 15:06:08 2003
@@ -26,6 +26,7 @@
 INSTALL_DOC = $(INSTALL) -m 644
 INSTALL_BIN = $(INSTALL) -m 755
 INSTALL_DIR = $(INSTALL) -m 755 -d
+DEST = 
 
 prefix		= @prefix@
 exec_prefix	= @exec_prefix@
@@ -85,14 +86,14 @@
 all:	$(BINS)
 
 install: $(BINS)
+	$(INSTALL_DIR) $(DESTDIR)$(sbindir)
 	for file in $(BINS); do \
 	strip $$file ;	\
-	$(INSTALL_BIN) $$file $(sbindir) ; \
+	$(INSTALL_BIN) $$file $(DESTDIR)$(sbindir) ; \
 	done	
-	$(INSTALL_DIR) $(sbindir)
-	$(INSTALL_DIR) $(mandir) $(mandir)/man1
+	$(INSTALL_DIR) $(DESTDIR)$(mandir) $(DESTDIR)$(mandir)/man1
 	for file in mtx.1 tapeinfo.1 scsitape.1 loaderinfo.1 ; do \
-	$(INSTALL_DOC) "$$file"  $(mandir)/man1 ;		  \
+	$(INSTALL_DOC) "$$file"  $(DESTDIR)$(mandir)/man1 ;	  \
 	done
 
 clean:
